actors/power/tests/power_actor_tests.rs

```diff
@@ -198,3 +201,176 @@ fn claimed_power_given_claim_does_not_exist_should_fail() {
     rt.verify();
     h.check_state();
 }
+
+fn get_enrolled_cron_ticks(rt: &MockRuntime, epoch: ChainEpoch) -> Vec<CronEvent> {
+    let state: State = rt.get_state();
+    let events_map = Multimap::from_root(
+        &rt.store,
+        &state.cron_event_queue,
+        CRON_QUEUE_HAMT_BITWIDTH,
+        CRON_QUEUE_AMT_BITWIDTH,
+    )
+    .expect("failed to load cron events");
+
+    let mut events: Vec<CronEvent> = Vec::new();
+    events_map
+        .for_each::<_, CronEvent>(&epoch_key(epoch), |_, v| {
+            events.push(v.to_owned());
+            Ok(())
+        })
+        .unwrap();
+
+    events
+}
+
+fn enroll_cron_event(
```

Comment:
The power actor is using the harness pattern to group method calls to the actor.  Please add this method to the harness